common/paramsbuilder/params.go

```diff
@@ -175,3 +177,34 @@ func (a APIModule) String() string {
 
 	return fmt.Sprintf("%s/%s", a.Label, a.Version)
 }
+
```

Comment:
This file is getting kind of big, maybe you can break it up into a couple of files? Maybe params.go, client.go, module.go, metadata.go, workspace.go

Reply:
I love files, will do. :laughing: